File: web/scripts/jstest.js

BABYLON.ShaderMaterialWithAlphaTestTexture = class extends BABYLON.ShaderMaterial {
    constructor(name, scene, shaderPath, options, alphaTexture) {
        super(name, scene, shaderPath, options);

        this.extension_alphaTestTexture = alphaTexture;
    }

    getAlphaTestTexture() {
        return this.extension_alphaTestTexture;
    }
}

File: web/scripts/jstest.dart
@JS("BABYLON")
library BabylonMaterialExtension;

import "package:CubeLib/CubeLib.dart";
import "package:js/js.dart";

@JS()
class ShaderMaterialWithAlphaTestTexture extends ShaderMaterial {
    external factory ShaderMaterialWithAlphaTestTexture(String name, Scene scene, dynamic shaderPath, [IShaderMaterialOptions options, Texture alphaTexture]);
}